tests/playwright/sanity/modules/floating-buttons/floating-elements-admin.test.ts

```diff
@@ -0,0 +1,209 @@
+			const visibleButtons = buttons.filter( { has: page.locator( ':visible' ) } );
+			await expect( visibleButtons ).toHaveCount( 3 );
+			await expect( advancedSettingsCustom ).toBeVisible();
+		} );
+
+		await test.step( 'Check that  our custom advanced tab has the five expected sections and no others.', async () => {
+			await advancedSettingsCustom.click();
+			const elementorControlsContainer = page.locator( '#elementor-controls' );
+
+			EditorSelectors.floatingElements.floatingButtons.controls.advanced.sections.forEach( async ( control ) => {
+				const element = elementorControlsContainer.locator( control );
+				await expect( element ).toBeVisible();
+			} );
+
+			const sections = elementorControlsContainer.locator( '.elementor-control-type-section' );
+			await expect( sections ).toHaveCount( 5 );
+		} );
+
+		await test.step( 'Check that there is no section to add widget.', async () => {
+			const addNewWidget = editor.getPreviewFrame().locator( EditorSelectors.addNewSection );
+			await expect( addNewWidget ).toBeHidden();
+		} );
+
+		await test.step(
+			'Check that we have a container wrapping our widget. There is only one action button, the Delete button. Clicking it should open the library modal.',
+			async () => {
+				const container = editor.getPreviewFrame().locator( EditorSelectors.container );
+				await expect( container ).toBeVisible();
+
+				const contactButtonWidget = container.locator( EditorSelectors.getWidgetByName( 'contact-buttons' ) );
+				await expect( contactButtonWidget ).toBeVisible();
+
+				const contactButtonActionsContainer = container.locator( '.elementor-editor-container-settings' );
+				const actions = contactButtonActionsContainer.locator( 'li' );
+				await expect( actions ).toHaveCount( 1 );
+
+				await contactButtonWidget.hover();
+				const deleteContainer = contactButtonActionsContainer.locator( '.elementor-editor-element-remove' );
+				await deleteContainer.click();
+				const libraryModal = page.locator( '#elementor-template-library-modal' );
+				await expect( libraryModal ).toBeVisible();
+			} );
+
+		let floatingElement, context, newPage;
+
+		await test.step( 'Verify that Setting as entire site displays the Widget on the FE. Check the snapshot for visual regression.', async () => {
+			await floatingElPage.goToFloatingButtonsPage();
+			const floatingElementTitle = page.locator( '.wp-list-table tbody tr td.page-title a.row-title' );
+			await floatingElementTitle.hover();
+			const setAsEntireSite = page.locator( '.row-actions .set_as_entire_site' );
+			await expect( setAsEntireSite ).toBeVisible();
+			await setAsEntireSite.click();
+			await page.waitForURL( '/wp-admin/edit.php?post_type=e-floating-buttons' );
+
+			context = await browser.newContext();
+			newPage = await context.newPage();
+			await newPage.goto( '/' );
+			floatingElement = newPage.locator( '.e-contact-buttons' );
+			await expect( floatingElement ).toBeVisible();
+
+			expect( await floatingElement.screenshot(
```

Comment:
I think I would add a few more screenshots, to ensure that the styling works as expected.